src/crypto/tls/handshake_messages.go

```diff
@@ -200,6 +202,19 @@ func (m *clientHelloMsg) marshal() []byte {
 					})
 				})
 			}
+			if m.delegatedCredentialSupported {
+				if len(m.supportedSignatureAlgorithmsDC) > 0 {
+					// Draft: https://tools.ietf.org/html/draft-ietf-tls-subcerts-09#section-4.1
+					b.AddUint16(extensionDelegatedCredentials)
+					b.AddUint16LengthPrefixed(func(b *cryptobyte.Builder) {
+						b.AddUint16LengthPrefixed(func(b *cryptobyte.Builder) {
+							for _, sigAlgo := range m.supportedSignatureAlgorithmsDC {
+								b.AddUint16(uint16(sigAlgo))
+							}
+						})
+					})
+				}
+			}
```

Comment:
Is this double-indented?

Reply:
Ah wait, it's just tabs.

I don't think I can resolve this myself on this repo :(

Reply:
Mmm.. what do you mean?

Reply:
It was a mistake, but I can't mark comments as resolved.

Github displays tabs as 8 spaces.

Reply:
Ah, ok. I'm closing this. All of this github problems might be gone when I squash prior to merge. Let's see.